### PR TITLE
Add random email extension

### PR DESCRIPTION
**Pull request by rkjaer**
## Description

A configurable extension for generating "random" (time-based) emails and copying/pasting them afterwards.

## Type of change

- New extension

## Screencast

CleanShot 2022-01-03 at 12 56 18
CleanShot 2022-01-03 at 13 10 56

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

File: extensions/random-email/package.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    {
+      "name": "generate-copy",
+      "title": "Copy random email",
+      "subtitle": "Random Email",
+      "description": "Copies a random email to the clipboard",
+      "mode": "no-view"
+    },
+    {
+      "name": "generate-paste",
+      "title": "Paste random email",
```
Comment (pernielsentikaer) [inline extensions/random-email/package.json:19]:
```suggestion
      "title": "Paste Random Email",
```

File: extensions/random-email/package.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  "name": "random-email",
+  "title": "Random email",
+  "description": "A quick way to create and copy a random email address",
+  "icon": "command-icon.png",
+  "author": "rkjaer",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "generate-copy",
+      "title": "Copy random email",
```
Comment (pernielsentikaer) [inline extensions/random-email/package.json:12]:
```suggestion
      "title": "Copy Random Email",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @rkjaer 🇩🇰 

Thanks for your contribution, I only have two small suggestion for you before I think we're ready to proceed 🔥 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Review by pernielsentikaer — APPROVED** [review]
Hi @rkjaer 👋 

Thanks for the update 🇩🇰 🔥 I'm going to accept it and then will my colleague do a final review before it's finally merged. Thanks again for your contribution 🚀

**Comment by rkjaer** [comment]
Thanks a lot @pernielsentikaer 🥳🇩🇰🍻

**Pull request merged**